cmd/fs-v1-multipart_test.go

```diff
@@ -20,15 +20,76 @@ import (
 	"bytes"
 	"path/filepath"
 	"testing"
+	"time"
 )
 
+func TestFSCleanupMultipartUploadsInRoutine(t *testing.T) {
+	// Prepare for tests
+	disk := filepath.Join(globalTestTmpDir, "minio-"+nextSuffix())
+	defer removeAll(disk)
+
+	obj := initFSObjects(disk, t)
+	fs := obj.(*fsObjects)
+
+	bucketName := "bucket"
+	objectName := "object"
+
+	obj.MakeBucketWithLocation(bucketName, "")
+	_, err := obj.NewMultipartUpload(bucketName, objectName, nil)
+	if err != nil {
+		t.Fatal("Unexpected err: ", err)
+	}
+
+	// Close the current active go-routine.
+	close(fs.staleMultipartUploadsDoneCh)
+
+	// Allocate a new done channel.
+	fs.staleMultipartUploadsDoneCh = make(chan struct{})
+
+	go fs.cleanupStaleMultipartUploadsRoutine(20*time.Millisecond, 0)
+
+	// Wait for 200ms such that we have given enough time for
+	// cleanup routine to kick in.
+	time.Sleep(40 * time.Millisecond)
```

Comment:
comment and code are not in sync. Either the comment should be 40ms or Sleep should take 200ms.